connect/cli/plugins/project/utils.py

```diff
@@ -35,6 +35,126 @@ def _run_hook_from_repo_dir(
     pass
 
 
+def eaas_introduction(config, idx, total):
+    intro = [
+        {
+            'name': 'introduction',
+            'type': 'input',
+            'message': 'Welcome to the Extension as a Service boilerplate utility.\n'
+            'You will drive through different screen dialogs where you provide required\n'
```

Comment:
Using this utility multiple data will be collected in order to create a project skeleton for your Connect Extension. This skeleton will contain all the needed things to start writing your code.